packages/core-app-api/src/app/AppManager.tsx

```diff
@@ -181,6 +181,10 @@ export class AppManager implements BackstageApp {
     this.apiFactoryRegistry = new ApiFactoryRegistry();
   }
 
+  getApis(): AnyApiFactory[] {
+    return Array.from(this.apis);
+  }
```

Comment:
We discussed this a bit more and figured it's best to made it only expose the api + deps, and not the factory function itself. Perhaps something like `getApiFactoryMetadata`, but it would make the API surface a bit unfortunately wide. Another option could be something like `getIntrospectionMetadata`, which in turn returns an object or instance where you get the `apiFactoryMetadata`. We can then merge that together with your routes idea and expose routing metadata there as well. What do you think?